### PR TITLE
Changes on Poem.js

### PR DESCRIPTION
**Pull request by jmuseyibli**
- Separate SVGs from Poem.js 
- Generalize styles.

### CODE REVIEW

File: components/svg/DayIcon.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import SvgUri from "react-native-svg-uri";
+export default () => (
+  <SvgUri
```
Comment (gulagha) [inline components/svg/DayIcon.js:4]: React-native-svg is more reliable in my opinion. Last update for react-native-svg-uri was 2 years ago.

File: components/svg/NightIcon.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import SvgUri from "react-native-svg-uri";
+export default () => (
+  <SvgUri
```
Comment (gulagha) [inline components/svg/NightIcon.js:4]: react-native-svg

File: components/svg/DayIcon.js (hunk 1 above)
Comment (jmuseyibli) [reply components/svg/DayIcon.js:4]: @gulagha as far as I know, react-native-svg-uri is a wrapper around react-native-svg. Please have a look: https://github.com/vault-development/react-native-svg-uri

Comment (gulagha) [reply components/svg/DayIcon.js:4]: @jmuseyibli that's right, but the react-native-svg version was 4.3.1 at that time, now it's 12.1.1

Comment (jmuseyibli) [reply components/svg/DayIcon.js:4]: Okay. I see your point. Gulagha, if you have time, please checkout this branch, and change the library.

Comment (gulagha) [reply components/svg/DayIcon.js:4]: Yep, sure

Comment (jmuseyibli) [reply components/svg/DayIcon.js:4]: Thanks!! :)

**Pull request merged**